database/test_data.py

```diff
@@ -479,12 +507,13 @@
         "createdBy": "",
         "isAllSectionsServiceLearning": True,
         "isPermanentlyDesignated": False,
-        "sectionBQuestion1": "",
-        "sectionBQuestion2": "",
-        "sectionBQuestion3": "",
-        "sectionBQuestion4": "",
-        "sectionBQuestion5": "",
-        "sectionBQuestion6": ""
+        # "sectionBQuestion1": "",
+        # "sectionBQuestion2": "",
+        # "sectionBQuestion3": "",
+        # "sectionBQuestion4": "",
+        # "sectionBQuestion5": "",
+        # "sectionBQuestion6": "",
+
```

Comment:
Don't comment things out that should just be removed.

Reply:
removed comments